flexget/plugins/output/prowl.py

```diff
@@ -52,7 +52,7 @@ def on_task_output(self, task, config):
             event = entry.get('event', config['event'])
             priority = entry.get('priority', config['priority'])
             description = config.get('description', entry['title'])
-            message_url = config.get('url')
+            message_url = config.get('url', '')
```

Comment:
That's not good enough dude. If the user supplied a url which is non renderable, like `{{ cannot_render_me }}` it'll still crash. You need to assign a value to `message_url` after catching the RenderError exception